docs/guides/how_to_guides/creating_and_editing_expectations/how_to_dynamically_load_evaluation_parameters_from_a_database.rst

```diff
@@ -28,6 +28,21 @@ Steps
         queries:
           current_genre_ids: "SELECT id FROM genres;"  # The query name and value can be replaced with your desired query
 
+    By default, query results will be returned as a list. If instead you need a scalar for your expectation, you can specify the return_type
+
+.. code-block:: yaml
+
+      my_query_store:
+        class_name: SqlAlchemyQueryStore
+        credentials: ${rds_movies_db}
+        queries:
+          current_ratings_max:
+            query: "SELECT MAX(rating) FROM ratings;"
+            return_type: "scalar"  # return_type can be either "scalar" or "list"
```

Comment:
Just a suggestion to clarify that the `return_type` can be omitted
```suggestion
            return_type: "scalar"  # return_type can be either "scalar" or "list" or omitted
```

Reply:
great idea - thanks!

Reply:
On second thought, if we use this type of notation for a query (multi-line instead of one line and dictionary vs. string) then return_type needs to be specifed.